controller/kubernetes/kubernetes.go

```diff
@@ -265,14 +256,16 @@ func (lbc *loadBalancerController) sync(key string) {
 		lbc.syncQueue.Requeue(syncAll, fmt.Errorf("deferring sync till endpoints controller has synced"))
 		return
 	}
+
+	// 1) apply new configs
 	requeue := false
-	cfgs, err := lbc.GetLBConfigs()
+	toApply, err := lbc.GetLBConfigs()
 	if err != nil {
 		log.Errorf("Error forming LB Config from Ingress spec: %v", err)
```

Comment:
do we need to requeue here as well and return?

Reply:
yup, will add